packages/@angular/cli/commands/init.run.ts

```diff
@@ -31,9 +35,14 @@ export default function initRun(commandOptions: any, rawArgs: string[]) {
 
   let npmInstall: any;
   if (!commandOptions.skipNpm) {
+    let packageManager = 'npm';
```

Comment:
I don't think you need this here, since `config.get(...)` will give you the default value.